core/src/snapshot/analyze.rs

```diff
@@ -1,5 +1,5 @@
 use crate::decompile::constants::AND_BITMASK_REGEX;
-
+use crate::snapshot::postprocessers::postprocess::cleanup;
```

Comment:
Rather than `postprocessers::postprocess`, you can probably get away with just doing `postprocess.rs` instead